apis/v1beta1/workspace_types.go

```diff
@@ -107,6 +107,10 @@ type WorkspaceParameters struct {
 	// +optional
 	Entrypoint string `json:"entrypoint"`
 
+	// Variable to show a plan on CR based on condition
```

Comment:
```suggestion
	// Include the output of terraform plan in the status.
	// The plan will be gzipped and base64 encoded.
```

This comment will show up as user-facing documentation, so it's important that it clearly describes what the field does and how it works.